packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -12,25 +12,36 @@
 # License for the specific language governing permissions and
 # limitations under the License.
 
+%if 0%{?amzn} >= 2
+%bcond_without systemd # with
+%else
+%bcond_with systemd # without
+%global running_semaphore %{_localstatedir}/run/ecs-init.was-running
+%endif
+%global _cachedir %{_localstatedir}/cache
+%global bundled_agent_version %{version}
+
 Name:           ecs-init
 Version:        1.18.0
-Release:        1%{?dist}
-Group:          System Environment/Base
-Vendor:         Amazon.com
+Release:        2%{?dist}
 License:        Apache 2.0
 Summary:        Amazon Elastic Container Service initialization application
-BuildArch:      x86_64
-BuildRoot:      %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+ExclusiveArch:  x86_64
```

Comment:
Why change from BuildArch to ExclusiveArch?

Reply:
https://fedoraproject.org/wiki/Packaging:Guidelines#Arch-Specific_Runtime_and_Build-Time_Dependencies